src/Sylius/Component/Grid/View/GridView.php

```diff
@@ -90,16 +91,40 @@ public function isSortedBy($fieldName)
     public function getSortingOrder($fieldName)
     {
         $this->assertFieldExists($fieldName);
+        Assert::true($this->isSortableBy($fieldName), sprintf('Field "%s" is not sortable.', $fieldName));
+
+        return $this->getCurrentlySortedBy()[$fieldName]['direction'];
+    }
+
+    /**
+     * @param string $fieldName
+     *
+     * @return bool
+     */
+    public function isSortedBy($fieldName)
+    {
+        $this->assertFieldExists($fieldName);
+        Assert::true($this->isSortableBy($fieldName), sprintf('Field "%s" is not sortable.', $fieldName));
 
-        return $this->getCurrentSorting()[$fieldName];
+        if ($this->parameters->has('sorting')) {
+            return array_key_exists($fieldName, $this->parameters->get('sorting'));
```

Comment:
Should null ``'sorting'`` be accepted? If no, we should use ``isset`` instead of ``array_ket_exists``.